README.md

```diff
@@ -657,14 +657,39 @@ map.pm.addControls({
 
 Reorder the buttons with
 ```js
-map.pm.Toolbar.changeControlOrder(['drawCircle', 'drawRectangle', 'removalMode', 'editMode'])
+map.pm.Toolbar.changeControlOrder(['drawCircle', 'drawRectangle', 'removalMode', 'editMode']);
 ```
 
 Receive the current order with
 ```js
-map.pm.Toolbar.getControlOrder()
+map.pm.Toolbar.getControlOrder();
 ```
 
+**Position**
+
+You can set the position of each block:
+
+```javascript
+map.pm.addControls({
+  positions: {
+    draw: 'topright',
+    edit: 'topleft',
+    options: undefined,
+    custom: undefined,
+    oneBlock: undefined,
```

Comment:
let's not encourage to pass `undefined` here. Remove the option completely from the example or pass `false`.